internal/image/build.go

```diff
@@ -188,3 +217,12 @@ func captureCommitSignOff(message string) []string {
 
 	return capturedSignatures
 }
+
+func GetAuthorization(source SignOffSource) (*signOffSignature, error) {
```

Comment:
Where will this function be used?

Reply:
https://github.com/hacbs-contract/ec-cli/pull/117/files#diff-c9cece346cc5ba89d52687fa6366ad4b759c7d9d35fa5b12953aa4f8fbd90fbcR99
https://github.com/hacbs-contract/ec-cli/pull/117/files#diff-faeeccd5a4f189b30736081616865d876bf4d0c70fd73ccebd1cc832736584c5R81